pkg/cli/info.go

```diff
@@ -1,12 +1,16 @@
 package cli
 
 import (
-	apiv1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
+	"encoding/json"
+	"reflect"
+
+	v1 "github.com/acorn-io/acorn/pkg/apis/api.acorn.io/v1"
```

Comment:
This is required. Go treats `/v*` as package versions. In order to reference this package as `v1`, this name is needed.

Reply:
> Go treats /v* as package versions

So IIUC, for a given package import, it's actually the top-level _module path_ that determines the major version of the module (and package) resolved. 

Barring any nested Modules, the _module path_ here is `github.com/acorn-io/acorn`, so that `v1` at the very end of the import statement shouldn't affect the version of the package that gets resolved.

but maybe I'm missing something

Reply:
No, you are absolutely right here. I just tested it and it seems fine to remove this alias.

There was an instance previously where such an alias was necessary. Not sure what the difference is. I will remove this one.